### ISSUE TITLE
Integrate scores with live website

### ISSUE THREAD

**Issue by trickeydan** [srobo/tasks#112]

**Comment by PeterJCLaw** [comment]
The https://github.com/srobo/competition-website repo handles this. I can't recall what API/stream it's pointing at currently (I wouldn't be surprised if it was the one on saffron), so I expect that that'll need changing.
We may also need to update the homepage it offers to support only having a single arena (it's built around the precedent of two arenas, though I did adjust it fairly easily for SourceBots last year to only have one).
I'm fairly sure that I updated its styling to match the new colour & logo earlier in the year, when we did the rest of the sub-sites, but that's worth double-checking.

Other than that, it _should_ be in a state where it can just be switched on. This is done by un-commenting some lines in the main website's `nginx.conf`: https://github.com/srobo/website/blob/master/_env/nginx.conf#L117-L132

**Comment by PeterJCLaw** [comment]
Oh, also the homepage which is part of https://github.com/srobo/competition-website has a slot for a livestream built-in, so #162 ought to be pretty easy -- assuming we're using youtube there's a line in the config to drop the new URL in: https://github.com/srobo/competition-website/blob/master/_config.yml#L19

Hrm, turns out there are some other config variables to update there too (search for `2017`).

**Comment by PeterJCLaw** [comment]
The API its pointed at is indeed saffron. We'll probably want to drop that in favour of a public facing compbox machine.
Either way, the config line of interest is: https://github.com/srobo/website/blob/master/_env/nginx.conf#L113-L115

**Comment by PeterJCLaw** [comment]
Blocked on having a public facing compbox (#233).

**Comment by PeterJCLaw** [comment]
While the underlying compstate (#119) doesn't yet have the real league schedule (#103), this is effectively done. I've checked that everything looks ok with a single arena and made a small fix on that.

For the moment I've used `saffron` as a temporary public API to unblock this, though as we already have #233 and https://github.com/srobo/website/pull/142 for resolving that I see little benefit in keeping this open now.
